app/src/main/AndroidManifest.xml

```diff
@@ -19,7 +19,8 @@
         tools:targetApi="31">
         <activity
             android:name=".MainActivity"
-            android:exported="true">
+            android:exported="true"
+            android:configChanges="orientation|screenSize|screenLayout|smallestScreenSize">
```

Comment:
We need to set `android:configChanges`, to prevent Android from recreating the activity, when the screen orientation changes. (Recreating the activity would result in reloading the webview, when the user enters fullscreen)

But Android wouldn't be Android, if `orientation` would be the only thing to set. We need to set the other change events as well, to work properly with foldable and dual screen devices. Tested on several Android simulators.